Demo/RxSwiftExtDemo/RxSwiftExtPlayground.playground/SupportCode.remap

```diff
@@ -0,0 +1,53 @@
+[
+ {
+  "file": "/Users/dkmadboe/Developer/Github/RxSwiftExt2/Demo/RxSwiftExtDemo/RxSwiftExtPlayground.playground/Sources/SupportCode.swift",
```

Comment:
There shouldn't be absolute paths in your playground -- unless this file should not be committed?